integration-tests/tests/update_worker_set.rs

```diff
@@ -285,14 +285,13 @@ fn worker_set_cannot_be_updated_again_while_pending_worker_is_not_yet_confirmed(
     );
 
     // try to rotate again. this should be ignored, because the first rotation is not yet confirmed
-    let second_worker_details = vec![("worker5".to_string(), 5)];
     let second_wave_of_new_workers =
-        test_utils::create_new_workers_vec(chains.clone(), second_worker_details);
+        test_utils::create_new_workers_vec(chains.clone(), vec![("worker5".to_string(), 5)]);
 
     let second_wave_session_id = test_utils::update_registry_and_construct_proof(
         &mut protocol,
-        &second_wave_of_new_workers,
-        &first_wave_of_new_workers, // as previous_workers compared to the second wave of new workers
+        &second_wave_of_new_workers, // as new_workers
+        &first_wave_of_new_workers, // as workers_to_remove
```

Comment:
same here, I'm confused about these comments. Are you trying to link these to the names of the parameters as defined? If that's the case don't do it. It's very brittle. If the name in the function changes you end up with comments that are both confusing and not linked to anything anymore. If it's hard to understand the arguments passed to a function, then it's better to refactor the function itself as I outlined in the standup

Reply:
Thank you for the guideline. I believe the function arguments are easy to understand (after we renamed the inputs to workers_to_remove and new_workers). I will remove these comments then.

Reply:
Addressed here: https://github.com/axelarnetwork/axelar-amplifier/pull/235/commits/7443afdaa1f02445a27893025f824272a6e4cfed